### PR TITLE
[15.0][IMP] sale_planner_calendar: New option to only update partner user_id if new_start is empty

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT51461

ping @sergio-teruel @CarlosRoca13

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]
Functional review OK

**Review by CarlosRoca13 — APPROVED** [review]

File: sale_planner_calendar/wizard/sale_planner_calendar_reassign.py [hunk 1]
```diff
@@ -106,6 +106,13 @@ def apply(self):
         for line in self.line_ids:
             if not line.new_user_id:
                 continue
+            if self.assign_new_salesperson_to_partner:
+                line.partner_id.with_context(
+                    skip_sale_planner_check=True
+                ).user_id = line.new_user_id
+            # If new_end is empty only update partner user_id
```
Comment (pedrobaeza) [inline sale_planner_calendar/wizard/sale_planner_calendar_reassign.py]:
```suggestion
            # If new_start is empty only update partner user_id
```

Comment (carlosdauden) [reply sale_planner_calendar/wizard/sale_planner_calendar_reassign.py]: Changed

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**